src/modules/core/components/TransactionList/TransactionDetails.tsx

```diff
@@ -268,7 +272,7 @@ const TransactionDetails = ({
     : OutgoingTransaction;
   return (
     <TransactionComponent
-      colony={colony}
+      colony={colony as ColonyType}
```

Comment:
For this kind of case, perhaps it's better to use:

```typescript
colony={colony!}
```

Because that way we don't have to specify the type again, only assert that it's defined.

Reply:
That said, it seems like there's an eslint rule we're using that would disallow this.

Reply:
This is actually an issue in the component itself. I fixed it by using a guard